cmd/ak/cmd/connections/delete.go

```diff
@@ -20,23 +18,16 @@ var deleteCmd = common.StandardCommand(&cobra.Command{
 		r := resolver.Resolver{Client: common.Client()}
 		c, id, err := r.ConnectionNameOrID(args[0])
 		if err != nil {
-			if errors.As(err, resolver.NotFoundErrorType) {
-				err = common.NewExitCodeError(common.NotFoundExitCode, err)
-			}
-			return err
+			return common.ToExitCodeError(err, "connection")
```

Comment:
I understand the motivation for the change here and under `!c.IsValid()` below, but I think it leads to a wrong result, cosmetically.

The expected output is "Error: connection not found" (if the problem is that the ID wasn't found), but `common.ToExitCodeError()` prints something more complicated (error's string + error message = unnecessary duplication).

And as a result, there's a legitimate failure in a system test. And because it's less user-friendly than before I wouldn't want to fix the test, but rather leave the error handling as-is.

We could definitely improve this systematically later, but it isn't worth the effort right now.

Reply:
I fixed a ToExitCodeError wrapper. Now it prints what was expected